app/main/views/guidance.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from flask.templating import render_template
+from ...main import main
+
+
+@main.route('/guidance/how-to-answer-supplier-questions')
+def answering_questions():
+    return render_template('guidance/answering_questions.html')
+
+
+@main.route('/guidance/how-to-shortlist-suppliers')
+def shortlisting_suppliers():
+    return render_template('guidance/shortlisting_suppliers.html')
+
+
+@main.route('/guidance/how-to-evaluate-suppliers')
+def evaluating_suppliers():
+    return render_template('guidance/evaluating_suppliers.html')
+
+
+@main.route('/guidance/how-marketplace-evaluated')
+def evaluated():
```

Comment:
These function names are in a global namespace.  I think the other names are okay, but this one's a bit short and doesn't mean anything out of context.  I suggest just making it "how_marketplace_evaluated" because it's not like we're typing it a lot, anyway.